packages/ui/src/components/editor/panels/Scenes/container/index.tsx

```diff
@@ -50,12 +50,16 @@ export default function ScenesPanel() {
   const contextMenuRef = useRef(null)
   const isContextMenuOpen = useHookstate<AssetType['id']>('')
   const scenesLoading = scenesQuery.status === 'pending'
-  const onCreateScene = async () => onNewScene()
 
-  const onClickScene = (scene: AssetType) => {
-    getMutableState(EditorState).scenePath.set(scene.assetURL)
+  const isCreatingScene = useHookstate(false)
+  const handleCreateScene = async () => {
+    isCreatingScene.set(true)
+    await onNewScene()
```

Comment:
we need to try catch this to ensure if onNewScene throws that we set it to false afterwards

Reply:
[actually `onNewScene` catches the error internally](https://github.com/EtherealEngine/etherealengine/blob/e4902c0394bdd155c3c72158a2eb46e58f5781f3/packages/editor/src/functions/sceneFunctions.tsx/#L134-L158)